### PR TITLE
ruff review anne

### PR DESCRIPTION
**Pull request by ahms5**
for #114

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -43,7 +43,7 @@ tests = [
     "pytest",
     "pytest-cov",
     "watchdog",
-    "ruff==0.4.1",
+    "ruff==0.8.2",
```
Comment (ahms5) [inline pyproject.toml:46]: this does just make the change in the contest.py

File: sofar/io.py [hunk 2]
```diff
@@ -212,11 +212,11 @@ def _read_netcdf(filename, verify, verbose, mode):
     if verify:
         try:
             sofa.verify(mode="read")
-        except: # noqa (No error handling - just improved verbosity)
+        except ValueError as e:
```
Comment (f-brinkmann) [inline sofar/io.py:215]: I usally do a bare except only I experienced multiple possible errors during debugging. Passing tests must not necessarily mean that this catches all possible errors. Maybe revert to be on the safe side and provide verbose feedback?

File: sofar/update_conventions.py [hunk 3]
```diff
@@ -337,9 +337,9 @@ def _convention_csv2dict(file: str):
             for ff, field in enumerate(fields):
                 convention[line[0]][field.lower()] = line[ff + 1]
 
-        except: # noqa
+        except ValueError as e:
```
Comment (f-brinkmann) [inline sofar/update_conventions.py:340]: See above.

File: tests/test_sofa.py [hunk 4]
```diff
@@ -91,25 +91,25 @@ def test_list_dimensions(capfd):
 
     # test FIR Data
     sofa = sf.Sofa("GeneralFIR")
-    sofa.list_dimensions    # noqa: B018
+    _ = sofa.list_dimensions
```
Comment (f-brinkmann) [inline tests/test_sofa.py:94]: Haha, why did I not think of this? Thx.

**Review by f-brinkmann — COMMENTED** [review]
Thanks for the detailed check. Made one question for discussion.

File: tests/test_sofa.py (hunk 4 above)
Comment (ahms5) [reply tests/test_sofa.py:94]: I came across it in the documentation; I hadn’t thought of it either! 😂

**Review by f-brinkmann — APPROVED** [review]

**Pull request merged**